src/util/googleMaps.js

```diff
@@ -47,9 +47,10 @@ export const getPlaceDetails = (placeId, sessionToken) =>
     const serviceStatus = window.google.maps.places.PlacesServiceStatus;
     const el = document.createElement('div');
     const service = new window.google.maps.places.PlacesService(el);
+    const fields = ['address_component', 'formatted_address', 'geometry', 'place_id'];
```

Comment:
Isn't the `place_id` known already?

Reply:
Yes, but it doesn't hurt to have that returned also (it's part of the basic fields => same pricing). Anyway, id's included to the return object makes them easier to be reused in Redux store if that kind of feature is needed.